dali/common.h

```diff
@@ -212,6 +212,10 @@ inline std::string to_string(const DALITensorLayout& layout) {
       return "NCHW";
     case DALI_NHWC:
       return "NHWC";
+    // case DALI_NFHWC:
```

Comment:
?

Reply:
I will have to unify this once the #281 is merged. Will remove from this one for now.